lib/shoulda/matchers/action_controller/flash_store.rb

```diff
@@ -34,10 +34,8 @@ def has_value?(expected_value)
             expected_value === actual_value
           end
         end
-
-        def empty?
-          flash.empty?
-        end
+        
+        delegate :empty?, to: :flash
```

Comment:
I think the problem will not be solved after a rebase. 

This is the error at the moment:

```
undefined method `delegate' for Shoulda::Matchers::ActionController::FlashStore:Class (NoMethodError)
```

I solved this after adding this line at the beginning of the file:

```
require 'active_support/core_ext/module/delegation'
```

I'm not sure if this is the best approach. What do you think, @mcmire?

Reply:
Hmmm weird. I wonder why that's happening. Perhaps we don't use `delegate` anywhere else. Anyway, your fix makes sense to me.

Reply:
@withshubh If you could add that require then I think that would fix CI.